app/datastructure/ds_methods.py

```diff
@@ -0,0 +1,125 @@
+from datetime import datetime as dt
+# from werkzeug.security import generate_hash_password, check_password_hash
+
+
+users = []
+messages = []
+class DSMethods:
```

Comment:
@Eubule , Good work man, could you kindly explain what `DSMethods` class  is.. we should use names that give a clear picture to what we are doing..